monitoring/angle-monitoring/src/test/java/com/farao_community/farao/monitoring/angle_monitoring/AngleMonitoringTest.java

```diff
@@ -146,20 +146,20 @@ public void testPreventiveStateOnly() {
         setUpCracFactory("network.xiidm");
         mockPreventiveState();
         runAngleMonitoring();
-        assertTrue(angleMonitoringResult.isUnsecure());
+        assertFalse(angleMonitoringResult.isUnsecure());
         angleMonitoringResult.getAppliedCras().forEach((state, networkActions) -> assertTrue(networkActions.isEmpty()));
-        assertEquals(angleMonitoringResult.printConstraints(), List.of("AngleCnec acPrev (with importing network element VL1 and exporting network element VL2) at state preventive has an angle of -211°."));
-        assertEquals(angleMonitoringResult.getAngle(acPrev, Unit.DEGREE), -211, ANGLE_TOLERANCE);
+        assertEquals(angleMonitoringResult.printConstraints(), List.of("All AngleCnecs are secure."));
+        assertEquals(angleMonitoringResult.getAngle(acPrev, Unit.DEGREE), -3.67, ANGLE_TOLERANCE);
     }
 
     @Test
     public void testCurativeStateOnlyWithNoRa() {
         setUpCracFactory("network.xiidm");
         mockCurativeStates();
         runAngleMonitoring();
-        assertTrue(angleMonitoringResult.isUnsecure());
+        assertFalse(angleMonitoringResult.isUnsecure());
```

Comment:
change mockCurative thresholds to keep insecure state : the idea is that the situation cannot be secured with no RA